samples/client/petstore/scala-sttp/src/main/scala/org/openapitools/client/api/UserApi.scala

```diff
@@ -0,0 +1,187 @@
+      .body(user)
+      .response(asJson[Unit])
+
+
+
+  /**
+   * This can only be done by the logged in user.
+   * 
+   * Expected answers:
+   *   code 400 :  (Invalid username supplied)
+   *   code 404 :  (User not found)
+   * 
+   * Available security schemes:
+   *   auth_cookie (apiKey)
+   * 
+   * @param username The name that needs to be deleted
+   */
+  def deleteUser(username: String)(implicit apiKey: ApiKeyValue): ApiRequestT[Unit] =
+    basicRequest
+      .method(Method.DELETE, uri"$baseUrl/user/${username}")
+      .contentType("application/json")
+      .cookie("AUTH_KEY", apiKey.value)
+      .response(asJson[Unit])
+
+
+
+  /**
+   * Expected answers:
+   *   code 200 : User (successful operation)
+   *   code 400 :  (Invalid username supplied)
+   *   code 404 :  (User not found)
+   * 
+   * @param username The name that needs to be fetched. Use user1 for testing.
+   */
+  def getUserByName(username: String): ApiRequestT[User] =
+    basicRequest
+      .method(Method.GET, uri"$baseUrl/user/${username}")
+      .contentType("application/json")
+      .response(asJson[User])
+
+
+
+  /**
+   * Expected answers:
+   *   code 200 : String (successful operation)
+   *              Headers :
+   *                Set-Cookie - Cookie authentication key for use with the `auth_cookie` apiKey authentication.
+   *                X-Rate-Limit - calls per hour allowed by the user
+   *                X-Expires-After - date in UTC when toekn expires
+   *   code 400 :  (Invalid username/password supplied)
+   * 
+   * @param username The user name for login
+   * @param password The password for login in clear text
+   */
+  def loginUser(username: String, password: String): ApiRequestT[String] =
+    basicRequest
+      .method(Method.GET, uri"$baseUrl/user/login?username=$username&password=$password")
+      .contentType("application/json")
+      .response(asJson[String])
+
+
+
+  /**
+   * Expected answers:
+   *   code 0 :  (successful operation)
+   * 
+   * Available security schemes:
+   *   auth_cookie (apiKey)
+   */
+  def logoutUser()(implicit apiKey: ApiKeyValue): ApiRequestT[Unit] =
+    basicRequest
+      .method(Method.GET, uri"$baseUrl/user/logout")
+      .contentType("application/json")
+      .cookie("AUTH_KEY", apiKey.value)
+      .response(asJson[Unit])
+
+
+
+  /**
+   * This can only be done by the logged in user.
+   * 
+   * Expected answers:
+   *   code 400 :  (Invalid user supplied)
+   *   code 404 :  (User not found)
+   * 
+   * Available security schemes:
+   *   auth_cookie (apiKey)
+   * 
+   * @param username name that need to be deleted
+   * @param user Updated user object
+   */
+  def updateUser(username: String, user: User)(implicit apiKey: ApiKeyValue): ApiRequestT[Unit] =
+    basicRequest
+      .method(Method.PUT, uri"$baseUrl/user/${username}")
+      .contentType("application/json")
+      .cookie("AUTH_KEY", apiKey.value)
+      .body(user)
+      .response(asJson[Unit])
+
```

Comment:
let's try to remove blank spaces (the moto is to think that the client was created by human rather than by machine..)

Reply:
optimised